aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md

```diff
@@ -1,202 +1,196 @@
 ---
 uid: signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc
-title: "Tutorial: Getting Started with SignalR 2 and MVC 5 | Microsoft Docs"
+title: "Real-time chat with SignalR 2 and MVC 5 | Microsoft Docs"
 author: pfletcher
-description: "This tutorial shows how to use ASP.NET SignalR 2 to create a real-time chat application. You will add SignalR to an MVC 5 application and create a chat view..."
+description: "This tutorial shows how to use ASP.NET SignalR 2 to create a real-time chat application. You will add SignalR to an MVC 5 application and create ..."
 ms.author: riande
-ms.date: 06/10/2014
+ms.date: 12/27/2018
 ms.assetid: 80bfe5fb-bdfc-41fe-ac43-2132e5d69fac
 msc.legacyurl: /signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc
 msc.type: authoredcontent
 ---
-Tutorial: Getting Started with SignalR 2 and MVC 5
-====================
-by [Patrick Fletcher](https://github.com/pfletcher), [Tim Teebken](https://github.com/timlt)
+
+# Tutorial: Real-time chat with SignalR 2 and MVC 5
+
+This tutorial shows how to use ASP.NET SignalR 2 to create a real-time chat application. You'll add SignalR to an MVC 5 application and create a chat view to send and display messages.
+
+In this tutorial you learn how to:
+
+> [!div class="checklist"]
+> * Set up the project
+> * Run the sample
+> * Examine the code
```

Comment:
you don't really "learn how to examine the code," you examine the code.  Could drop "learn how to" from the intro sentence